### PR TITLE
update ruamel dependency

### PR DESCRIPTION
**Pull request by itdependsnetworks**

### CODE REVIEW

**Comment by itdependsnetworks** [comment]
This is failing due to issue #695 not because of any changes made here.

File: pyproject.toml [hunk 1]
```diff
@@ -25,7 +25,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.6"
-"ruamel.yaml" = "^0.16"
+"ruamel.yaml" = ">= 0.16, <0.18"
```
Comment (dbarrosop) [inline pyproject.toml:28]: Why `<0.18`?

Comment (itdependsnetworks) [reply pyproject.toml:28]: I did not want to be over permissive

Comment (itdependsnetworks) [reply pyproject.toml:28]: Let me elaborate, I did not want to be so permissive not knowing your initial intention

**Comment by ktbyers** [comment]
@itdependsnetworks Can you rebase and update your poetry.lock file (to get David's mypy fix).

Also we should just do:

```
"ruamel.yaml" = "0.16.*"
```

That should allow 0.16 and greater, but disallow 1.x.x.

Alternatively we could just do >=0.16

**Comment by itdependsnetworks** [comment]
I went with `Alternatively we could just do >=0.16`, since semantic versioning technically treats every "minor" version as potentially non-backwards compatible, and poetry respects that.

**Review by dbarrosop — APPROVED** [review]

**Comment by dbarrosop** [comment]
Thanks!

**Pull request merged**